packages/react/src/Pagination/Pagination.test.tsx

```diff
@@ -98,6 +98,18 @@ describe('Pagination', () => {
     expect(screen.getByText('5')).not.toHaveAttribute('aria-current', 'true')
   })
 
+  it('render custom labels for the "previous" and "next" buttons', () => {
+    render(<Pagination totalPages={10} previousLabel="previous" nextLabel="next" />)
+    expect(screen.getByText('previous')).toBeInTheDocument()
+    expect(screen.getByText('next')).toBeInTheDocument()
```

Comment:
These could be more precise as well.

Reply:
Somehow this does not work??
```jsx
it('render custom labels for the "previous" and "next" buttons', () => {
    render(<Pagination totalPages={10} previousLabel="previous" nextLabel="next" />)
    const previousButton = screen.getByRole('button', { name: 'previous' })
    const nextButton = screen.getByRole('button', { name: 'next' })

    expect(previousButton).toBeInTheDocument()
    expect(nextButton).toBeInTheDocument()
  })
  ```

Reply:
The button's accessible name is the value of its `aria-label`; see [list item 2C here](https://www.w3.org/TR/accname-1.1/#mapping_additional_nd_te).

So we should probably use this:

```js
const previousButton = screen.getByRole('button', { name: 'Vorige pagina' })
const nextButton = screen.getByRole('button', { name: 'Volgende pagina' })

expect(previousButton).toHaveTextContent('previous')
expect(nextButton).toHaveTextContent('next')
```